microprofile/tests/arquillian/pom.xml

```diff
@@ -70,5 +64,9 @@
             <groupId>org.testng</groupId>
             <artifactId>testng</artifactId>
         </dependency>
+        <dependency>
```

Comment:
Why does it depend on MP metrics?

Reply:
Because of `HelidonDeployableContainer#cleanupBaseMetrics`